core/admin/Dockerfile

```diff
@@ -1,17 +1,18 @@
-FROM alpine:3.8
+FROM alpine:3.10
```

Comment:
Maybe we could use an ARG for these so we can mimic the behavior from the arm PR.